### ISSUE TITLE
'Too few arguments to function ...' when querying entry fieldValue.

### ISSUE THREAD

**Issue by ajmar** [harness-software/wp-graphql-gravity-forms#8]
I am querying for a specific form entry (though this behavior occurs when using gravityFormsEntries), and if I query for the EmailField fieldValue, null is simply returned and an internal server error occurs, with the debug message: `Too few arguments to function WPGraphQLGravityForms\\Types\\Union\\ObjectFieldValueUnion::WPGraphQLGravityForms\\Types\\Union\\{closure}(), 1 passed and exactly 3 expected`. The form only contains the EmailField, double checked that the entry exists. Cheers.

Query:
```
{
  gravityFormsEntry(id: "R3Jhdml0eUZvcm1zRW50cnk6Mw==") {
    formId
    ip
    fields {
      edges {
        node {
          ... on EmailField {
            id
            type
            label
          }
        }
        fieldValue {
          ... on EmailFieldValue {
            value
          }
        }
      }
    }
  }
}
```

Query result:
```
{
  "errors": [
    {
      "debugMessage": "Too few arguments to function WPGraphQLGravityForms\\Types\\Union\\ObjectFieldValueUnion::WPGraphQLGravityForms\\Types\\Union\\{closure}(), 1 passed and exactly 3 expected",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 12,
          "column": 9
        }
      ],
      "path": [
        "gravityFormsEntry",
        "fields",
        "edges",
        0,
        "fieldValue"
      ]
    }
  ],
  "data": {
    "gravityFormsEntry": {
      "fields": {
        "edges": [
          {
            "node": {
              "id": 2,
              "type": "email",
              "label": "Email"
            },
            "fieldValue": null
          }
        ]
      },
      "ip": "10.48.210.1",
      "formId": 2
    }
  }
}
```

**Comment by ajmar** [comment]
Resolved with #9.

**Comment by kellenmace** [comment]
Nice! Thanks for catching this and  submitting a PR, @ajmar 👍. Just got it merged in.
